jobs/e2e_node/containerd/containerd-master/image-config.yaml

```diff
@@ -4,6 +4,6 @@ images:
     project: ubuntu-os-gke-cloud
     metadata: "user-data</go/src/github.com/containerd/cri/test/e2e_node/init.yaml,containerd-configure-sh</go/src/github.com/containerd/cri/cluster/gce/configure.sh,containerd-env</workspace/test-infra/jobs/e2e_node/containerd/containerd-master/env"
   cos-stable:
-    image_regex: cos-stable-73-11647-510-0
+    image_family: cos-stable
```

Comment:
I would suggest to use cos-81-lts rather than cos-stable here.

Reply:
kind of raises the question of what lts vs stable means. If it's a floating image it's a floating image. I do not know why this runs against both ubuntu and cos.
/shrug

Reply:
Maybe both ubuntu and cos are OSes used for early K8s development.